lib/bolt/outputter/human.rb

```diff
@@ -1,27 +1,80 @@
 module Bolt
   class Outputter
     class Human < Bolt::Outputter
+      COLORS = { red: "31",
```

Comment:
Do these get translated on Windows? Or does this only work in *nix?

Reply:
presumably the latter